packages/hash/frontend/src/components/layout/PageHeader/HeaderIconButton.tsx

```diff
@@ -1,18 +1,22 @@
 import { styled } from "@mui/material";
 import { IconButton } from "../../IconButton";
 
+const focusBorderOffset = 6;
+const focusBorderWidth = 3;
+
 export const HeaderIconButton = styled(IconButton)<{ open: boolean }>(
-  ({ theme, open }) => ({
-    boxShadow: open
-      ? `0px 0px 0px 2px ${theme.palette.common.white}, 0px 0px 0px 5px ${theme.palette.gray[40]}`
-      : "unset",
```

Comment:
@akash-joshi, I removed this because I noticed the gray ring isn't meant to show up when the dropdown is active in the designs.